test/token/ERC7984/extensions/ERC7984Rwa.test.ts

```diff
@@ -708,7 +710,7 @@ describe('ERC7984Rwa', function () {
       it(`should not transfer if ${arg ? 'sender' : 'receiver'} blocked `, async function () {
         const { token, admin: manager, recipient, anyone } = await fixture();
         const account = arg ? recipient : anyone;
-        await token.$_setCompliantTransfer();
+        await token.$_setCompliant();
         const encryptedInput = await fhevm
           .createEncryptedInput(await token.getAddress(), recipient.address)
           .add64(25)
```

Comment:
@arr00
> compliance modules be like the IERC7984RwaTransferComplianceModule

Yes, I like that too 👍 
> post transfer function

Yes, see next commit [Add post transfer hook](https://github.com/OpenZeppelin/openzeppelin-confidential-contracts/pull/197/commits/86d5250216bd1617376026f80a5e869eb4562b7a).

> which follows ERC3643. There is a pretransfer check

Can you point me a reference to pre checks in modules? To me `isCompliantTransfer(..)` _is_ the non-mutating pre check.